sktime/detection/ggs.py

```diff
@@ -374,54 +374,32 @@ class GreedyGaussianSegmentation(BaseDetector):
     multivariate time series into segments, where the data in each segment
     could be modeled as independent samples from a multivariate Gaussian
     distribution. It uses a dynamic programming search algorithm with
-    a heuristic that allows finding approximate solution in linear time with
-    respect to the data length and always yields locally optimal choice.
-
-    Greedy Gaussian Segmentation (GGS) fits a segmented gaussian model (SGM)
-    to the data by computing the approximate solution to the combinatorial
-    problem of finding the approximate covariance-regularized  maximum
-    log-likelihood for fixed number of change points and a reagularization
-    strength. It follows an interactive procedure
-    where a new breakpoint is added and then adjusting all breakpoints to
-    (approximately) maximize the objective. It is similar to the top-down
-    search used in other change point detection problems.
+    a heuristic that allows finding approximate solutions in linear time with
+    respect to the data length and always yields a locally optimal choice.
 
     Parameters
     ----------
     k_max: int, default=10
         Maximum number of change points to find. The number of segments is thus k+1.
-    lamb: : float, default=1.0
+    lamb: float, default=1.0
         Regularization parameter lambda (>= 0), which controls the amount of
-        (inverse) covariance regularization, see Eq (1) in [1]_. Regularization
-        is introduced to reduce issues for high-dimensional problems. Setting
-        ``lamb`` to zero will ignore regularization, whereas large values of
-        lambda will favour simpler models.
+        (inverse) covariance regularization. A higher lambda favors simpler models.
     max_shuffles: int, default=250
         Maximum number of shuffles
     verbose: bool, default=False
-        If ``True`` verbose output is enabled.
+        If True, verbose output is enabled.
     random_state: int or np.random.RandomState, default=None
-        Either random seed or an instance of ``np.random.RandomState``
+        Either random seed or an instance of np.random.RandomState
```

Comment:
What is the motivation for removing the ``? The backticks make the text inside render as an inline literal which I think is useful for differentiating be regular text and code. You can see how this renders here https://www.sktime.net/en/v0.24.1/api_reference/auto_generated/sktime.annotation.ggs.GreedyGaussianSegmentation.html.

Reply:
Adressed in 5eb08a4